src/sinol_make/structs/status_structs.py

```diff
@@ -15,3 +17,88 @@ def __str__(self):
 
     def __repr__(self):
         return self.name
+
+    @staticmethod
+    def from_str(status):
+        if status == "CE":
+            return Status.CE
+        elif status == "TL":
+            return Status.TL
+        elif status == "ML":
+            return Status.ML
+        elif status == "RE":
+            return Status.RE
+        elif status == "WA":
+            return Status.WA
+        elif status == "OK":
+            return Status.OK
+        else:
+            return Status.PENDING
```

Comment:
is there a `status` string that means `PENDING`? Maybe we should warn/error when the content of `status` doesn't make sense?